js/components/featuredProjectsList.js/featuredProjectsList.js

```diff
@@ -0,0 +1,20 @@
+function featuredProjectsList(selector, data) {
+  const DOM = document.querySelector(selector);
+  let html = '';
+  for (const item of data) {
+    html += `<div class="col-12 col-sm-4 latest-projects-block">
+        <div class="project-img">
+        <img src="../../../img/project-img${item.image}.webp" alt="Project" class="sizing" />
```

Comment:
Nors kodo nepasileidau, bet kažkaip abejoje, jog reikia tiek daug ../ 🤔

Reply:
too much relative path updated. Aciu

Reply:
o kaip imanoma iseiti is triju folderiu trumpesniu budu? cia dar kazkokios kitokios taisykles veikia?

Reply:
nes pavyzdziui mano servery tai dabar isvis be relative path viskas veikia, o ikelus i githaba ar suveiks visad?

Reply:
Po kolkas veikia, taciau ar veiks ikelus i github master nezinia. I si klausima tikriausiai gali atsakyti @belauzas